packages/slice-machine/server/src/api/state.ts

```diff
@@ -105,7 +107,21 @@ export default async function handler(
     prismicAPIUrl: prismicData.base,
   };
 
-  if (serverState.libraries) req.tracker.libraries(serverState.libraries);
+  /*
+   * TEMPORARY
+   * Today, we need to have access to the cookie to get the anonymousID from segment but
+   * we want to log that event only at start time. Because of the necessity to access the cookies,
+   * we can't put that code on the start script just yet.
+   * We need to define a potential alternative for this event.
+   * In the meantime, this flag prevent the event flood
+   */
+  if (temp_first_start_flag && serverState.libraries) {
```

Comment:
If this is meant to be send only once, shouldn't it be send on the front-end ?
I didn't saw yet where the anonymousId is define but I think this would fit on a useEffect when the state is define for the first time.

Reply:
We need to do it on server side to be able to parse the manifest and enrich the event.
The best way would be to do it in the start script but then we wouldn't have access to the anonymousID stored in the cookies